src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/participationhistory/restapi/CourseParticipationIntegrationTest.kt

```diff
@@ -324,9 +331,16 @@ constructor(
       .accept(MediaType.APPLICATION_JSON)
       .exchange()
       .returnResult<Any>().status
+
```

Comment:
There's a few bits changing in this commit, though I appreciate they're making the tests a bit more readable. Could you add a bit more to the commit message just to document this? Don't need to document every single method, just more of an "and add some test helpers to improve readability".

While you're at it, would you specify this is about course participations and the name field e.g: "Update course participation factory & associated unit/integration tests now that we've added the `name` field"